CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * Added a validation to ensure integrations are not skipped and at least one test playbook is not skipped for each integration or script.
 * Added to the Content Tests support for `context_print_dt`, which queries the incident context and prints the result as a json.
 * Added new validation for the `xsoar_config.json` file in the **validate** command.
+* Added a version differences section to readme in **generate-docs** command.
```

Comment:
please note the 2 additional flags as well.

Reply:
done.